src/melee/ft/chara/ftNess/ftNess_AttackHi4.c

```diff
@@ -129,27 +122,28 @@ s32 ftNess_YoyoCheckEnvColl(
 
     func_8004730C(&sp34, &sp1C); // EnvironmentCollisionCheck_NessYoYo
 
-    // these might be bitfields
+    /// @todo Define flags.
     retval = 0;
-    if ((sp34.x134_envFlags & 0x18000) != 0) {
+    if ((sp34.x134_envFlags & 0x18000) != 0)
         retval |= 0x8000;
-    }
-    if ((sp34.x134_envFlags & 0x3F) != 0) {
+
+    if ((sp34.x134_envFlags & 0x3F) != 0)
         retval |= 1;
-    }
-    if ((sp34.x134_envFlags & 0xFC0) != 0) {
+
+    if ((sp34.x134_envFlags & 0xFC0) != 0)
         retval |= 0x40;
-    }
-    if ((sp34.x134_envFlags & 0x6000) != 0) {
+
+    if ((sp34.x134_envFlags & 0x6000) != 0)
         retval |= 0x2000;
-    }
+
     return retval;
 }
 
+#ifdef MUST_MATCH
+#pragma push
 #pragma dont_inline on
+#endif
```

Comment:
Would be nice if there were a way to make a MATCHING_DONT_INLINE macro for this. Modern compilers have a `_Pragma` expression that can expand from macros but I don't think it works with MWCC.

Reply:
(not a request for change, just a side remark)